packages/wallets/src/components/DerivAppsSection/DerivAppsSection.scss

```diff
@@ -9,17 +9,7 @@
     gap: 1.6rem;
 
     @include mobile {
-        min-width: 32.8rem;
-    }
-
-    &__get-account {
```

Comment:
@aizad-deriv if you delete this class description in this case you have to delete `wallets-deriv-apps-section__get-account` from `className` in `DerivAppsGetAccount` component. And also I'm pretty sure we need to set `padding` to 0 for `DerivAppsGetAccount` to be consistent with the design

Reply:
I see but by setting it to 0 it will too big of a gap don't you think?

Reply:
![localhost_8443_wallets (2)](https://github.com/binary-com/deriv-app/assets/103104395/74e8d3ae-66d2-4e14-87f1-b608e623a96a)